cmd/defaults.go

```diff
@@ -54,6 +57,84 @@ func GetConfig() *ToolConfig {
 	return &config
 }
 
+func GetCwdStoresRoot() string {
+	cwd, err := os.Getwd()
+	if err != nil {
+		return ""
+	}
+	dir := cwd
+	ok, err := isOperatorDir(dir)
+	if err != nil {
+		return ""
+	}
+	if ok {
+		return filepath.Dir(dir)
+	}
+
+	// search down
+	infos, err := ioutil.ReadDir(dir)
```

Comment:
This is scary, what if there is more than one down there, you just take the first one? I was thinking we only go up

Reply:
all we are determining here is there's an operator directory right below.
The code for selection is:
```	switch len(choices) {
	case 0:
		return ""
	case 1:
		return choices[0]
	default:
		for _, v := range choices {
			if v == current {
				return current
			}
		}
		return ""
	}```

Reply:
If there's one, that it is selected, otherwise, it is only set if it matches a value calculated.